static/js/app.js

```diff
@@ -847,6 +864,101 @@ $(function () {
             this.fetchMarkers();
             this.updateFilterString();
         },
+        loadFilterFromParameters: function() {
+            if (this.show_discussions=='1') {$("#checkbox-discussions").prop("checked", true)} else {$("#checkbox-discussions").prop("checked", false)}
```

Comment:
Again, it would be good to avoid repeating yourself. Maybe:

    BOOL_ATTRS.forEach(function(attr) {
        if (this[attr]=='1') {
            ...
        }
    });